### PR TITLE
[1.14] WIP Integration fixes

### PR DESCRIPTION
**Pull request by haircommander**
some shot in the dark integration test fixes because the errors in https://github.com/cri-o/cri-o/pull/2605 confuse me

### CODE REVIEW

**Comment by haircommander** [comment]
ah, closing in favor of #2607

File: test/network.bats [hunk 1]
```diff
@@ -182,7 +182,7 @@ function teardown() {
 	# ensure that the server cleaned up sandbox networking if the sandbox
 	# failed after network setup
 	rm -f /var/lib/cni/networks/crionet_test_args/last_reserved_ip*
-	num_allocated=$(ls /var/lib/cni/networks/crionet_test_args | wc -l)
+	num_allocated=$(ls /var/lib/cni/networks/crionet_test_args | grep -v lock | wc -l)
```
Comment (saschagrunert) [inline test/network.bats]: Ah, im wondering if we still need these changes...

Comment (haircommander) [reply test/network.bats]: Yeah your PR will also need this i think

Comment (saschagrunert) [reply test/network.bats]: Added it, thanks 🙏

**Pull request closed without merging**